### PR TITLE
feat: added homepage

### PR DESCRIPTION
**Pull request by Radheshyam1234**
<hr/>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    <link rel="stylesheet" href="./index.css" />
+    <title>Front-Quiz</title>
+  </head>
+  <body>
+    <header class="header">
+      <i class="fas fa-bars hamburger-menu"></i>
+
+      <a href="#" class="logo"><i class="fas fa-book-open"></i>Front-Quiz</a>
+      <div class="nav-links">
+        <a href="" class="nav-link">Home</a>
```
Comment (Hamdrive) [inline index.html:17]: add a `#` here as well

File: index.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+              loading="lazy"
+              src="./assets/Css grid-image.png"
+              alt="Nature"
+              class="img-responsive"
+            />
+          </div>
+          <h5 class="text-center category-title">Flexbox quiz</h5>
+          <a href="./quiz-rules.html">
+            <button class="btn secondary-btn">Start</button></a
+          >
```
Comment (Hamdrive) [inline index.html:134]: Consider tweaking your Prettifier file to format properly

**Review by Hamdrive — APPROVED** [review]
Good work done, have suggested some code formatting fixes which can be easily done
Also got to learn about lazy loading attribute with images!

**Review by theprogrammerinyou — APPROVED** [review]
Good job done.

**Pull request merged**